charts/fybrik/integration-tests.no-cluster-scope.values.yaml

```diff
@@ -0,0 +1,65 @@
+clusterScoped: false
```

Comment:
Do we need this entire file? can we just use the flag `--set` for the `helm install` command to overwrite the values ?

Reply:
I think its cleaner to have the entire file, otherwise, we need to add --set to the helm install in https://github.com/fybrik/fybrik/blob/master/Makefile#L44 (and we do not always add new settings)